frontend/src/app/choreography/groups/four-group/four-group.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
**chore:** delete these empty tests (you can disable the automatic creation too)